README.md

```diff
@@ -13,24 +13,16 @@ log.SetOutput(r)
 
 It can also be configured through a YAML file.
 
-Currently, due to a technical limitation, if a key is unknown, it will
+Currently, due to a technical limitation, if a key is unknown it will
 just be ignored. Be extra-careful that all keys are correctly spelled.
 
 ## Configuration
 
-The reporter encompasses the following aspects, each of them having its
-own subsection:
-
- - ``logging``: `Logging`_,
- - ``metrics``: `Metrics`_,
- - ``sentry``: `Crash reporting`_.
- - ``prefix``: `Prefix`_.
-
 ### Logging
 
 Here is an example configuration:
 
-```yaml
+``yaml
```

Comment:
Aren't you missing a  "`" sign on opening and closing?

Reply:
Good catch, missed this one. Fixed!